remote_app/manifest.webapp

```diff
@@ -28,6 +28,15 @@
       "description": "Dongle App"
     }
   },
+  "activities": {
+   "share": {
+      "filters": {
+        "type": "url"
+      },
+      "disposition": "window",
```

Comment:
I would like to have an `inline` instead. We would like to not change the context of the user.